ckanext/harvest/tests/test_action.py

```diff
@@ -253,12 +253,15 @@ def test_create(self):
 
         for key in source_dict.keys():
             assert_equal(source_dict[key], result[key])
-
+            
         # Check that source was actually created
         source = harvest_model.HarvestSource.get(result['id'])
         assert_equal(source.url, source_dict['url'])
         assert_equal(source.type, source_dict['source_type'])
 
+        # New source is created so the harvest_log table should be populated
+        assert model.Session.query(harvest_model.HarvestLog).count() > 0
```

Comment:
Just testing existence tends to bite you later as your tests get more complex. Instead of checking the length of the results, look for and compare the actual `HarvestLog` result that you expected to be stored.